### PR TITLE
Add a new BTag counter for DeepCSV

### PR DESCRIPTION
**Pull request by aperloff**

### CODE REVIEW

File: TreeMaker/python/doZinvBkg.py [hunk 1]
```diff
@@ -75,7 +75,7 @@ def reclusterZinv(self, process, cleanedCandidates, suff):
     from PhysicsTools.PatAlgos.tools.jetTools import addJetCollection
     jecLevels = ['L1FastJet', 'L2Relative', 'L3Absolute']
     if self.residual: jecLevels.append("L2L3Residual")
-    btagDiscs = ['pfCombinedInclusiveSecondaryVertexV2BJetTags']
+    btagDiscs = ['pfCombinedInclusiveSecondaryVertexV2BJetTags','pfDeepCSVDiscriminatorsJetTags:BvsAll']
```
Comment (kpedro88) [inline TreeMaker/python/doZinvBkg.py:78]: is this the correct way to rerun DeepCSV in jet toolbox?

Comment (aperloff) [reply TreeMaker/python/doZinvBkg.py:78]: I did a test and it looks to me like it's being re-run. I made sure that in general the BTagsclean and BTagsDeepCSVclean branches are similar. I also checked that there are a couple of differences to make sure one branch wasn't just a copy of the other. However, I didn't see any explicit directions to "re-run" DeepCSV in the JetToolbox.

```
root [2] PreSelection->Scan("BTags:BTagsclean:BTagsDeepCSV:BTagsDeepCSVclean")
************************************************************
*    Row   *     BTags * BTagsclea * BTagsDeep * BTagsDeep *
************************************************************
*        0 *         1 *         1 *         1 *         1 *
*        1 *         1 *         1 *         1 *         1 *
*        2 *         2 *         0 *         2 *         0 *
*        3 *         1 *         0 *         2 *         0 *
*        4 *         1 *         1 *         1 *         1 *
*        5 *         1 *         0 *         1 *         0 *
*        6 *         1 *         0 *         1 *         0 *
*        7 *         3 *         0 *         2 *         0 *
*        8 *         2 *         0 *         2 *         0 *
*        9 *         1 *         0 *         0 *         0 *
*       10 *         1 *         1 *         1 *         1 *
*       11 *         2 *         0 *         1 *         0 *
*       12 *         1 *         1 *         1 *         1 *
*       13 *         2 *         2 *         2 *         2 *
*       14 *         2 *         2 *         2 *         2 *
*       15 *         2 *         2 *         2 *         2 *
*       16 *         1 *         0 *         0 *         0 *
*       17 *         2 *         2 *         2 *         2 *
*       18 *         0 *         0 *         0 *         0 *
*       19 *         1 *         1 *         1 *         1 *
*       20 *         0 *         0 *         0 *         0 *
*       21 *         3 *         0 *         3 *         0 *
*       22 *         0 *         0 *         0 *         0 *
*       23 *         0 *         0 *         0 *         0 *
*       24 *         1 *         1 *         1 *         1 *
Type <CR> to continue or q to quit ==>
*       25 *         2 *         0 *         2 *         0 *
*       26 *         1 *         1 *         1 *         1 *
*       27 *         2 *         0 *         2 *         0 *
*       28 *         1 *         1 *         1 *         1 *
*       29 *         2 *         0 *         2 *         0 *
*       30 *         2 *         2 *         2 *         2 *
*       31 *         0 *         0 *         0 *         0 *
*       32 *         1 *         0 *         1 *         0 *
*       33 *         0 *         0 *         0 *         0 *
*       34 *         2 *         0 *         2 *         0 *
*       35 *         1 *         1 *         1 *         1 *
*       36 *         1 *         0 *         1 *         0 *
*       37 *         3 *         0 *         2 *         0 *
*       38 *         1 *         1 *         0 *         0 *
*       39 *         2 *         0 *         2 *         0 *
*       40 *         1 *         0 *         1 *         0 *
*       41 *         1 *         0 *         1 *         0 *
*       42 *         2 *         2 *         2 *         2 *
*       43 *         1 *         1 *         1 *         1 *
*       44 *         0 *         0 *         1 *         0 *
*       45 *         1 *         1 *         1 *         1 *
*       46 *         1 *         0 *         1 *         0 *
*       47 *         2 *         0 *         2 *         0 *
*       48 *         1 *         1 *         1 *         1 *
*       49 *         3 *         3 *         3 *         3 *
Type <CR> to continue or q to quit ==> q
************************************************************
(long long) 50
root [3] PreSelection->Scan("BTags:BTagsclean:BTagsDeepCSV:BTagsDeepCSVclean","BTagsclean!=BTagsDeepCSVclean")
************************************************************
*    Row   *     BTags * BTagsclea * BTagsDeep * BTagsDeep *
************************************************************
*       38 *         1 *         1 *         0 *         0 *
*       67 *         1 *         1 *         0 *         0 *
*       73 *         3 *         3 *         2 *         2 *
*       96 *         1 *         1 *         0 *         0 *
************************************************************
```

Comment (kpedro88) [reply TreeMaker/python/doZinvBkg.py:78]: Huh, that's smarter/easier than I would have expected.

Comment (aperloff) [reply TreeMaker/python/doZinvBkg.py:78]: Also this isn't using the jet toolbox. it's using the PAT tool addJetCollection.

Comment (kpedro88) [reply TreeMaker/python/doZinvBkg.py:78]: Ah, that's probably better supported then.

**Pull request merged**